certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/ag7804/AG7804CertificateActionSpecification.java

```diff
@@ -0,0 +1,136 @@
+package se.inera.intyg.certificateservice.infrastructure.certificatemodel.ag7804;
+
+import java.util.List;
+import se.inera.intyg.certificateservice.domain.action.certificate.model.CertificateActionType;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.CertificateActionSpecification;
+import se.inera.intyg.certificateservice.domain.common.model.Role;
+
+public class AG7804CertificateActionSpecification {
+
+  private AG7804CertificateActionSpecification() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  public static List<CertificateActionSpecification> create() {
```

Comment:
Look over the actions. We have differences from FK7804. Are you sure FMB and SRS should be active? Also the messages/complement shouldn't be there:

Ärendekommunikation ska inte vara tillgängligt

Kompletteringsbegäran ska inte vara tillgängligt